src/containers/shared/utils.js

```diff
@@ -139,6 +139,18 @@ export const localizeDate = (date, lang = 'en-US', options = {}) => {
   return new Intl.DateTimeFormat(lang, options).format(date);
 };
 
+export const concatTx = (a, b) => {
+  if (a.length === 0) return b;
```

Comment:
Add some comments explaining what this function does/what it's used for, and rename `a` and `b` to be more descriptive